pontoon/base/tests/__init__.py

```diff
@@ -95,12 +99,18 @@ class Meta:
         model = ChangedEntityLocale
 
 
+# This helps to test translation from different time periods.
+PERIOD_SIZE = 14 # months
```

Comment:
The custom logic for creating Translations at specific dates should not be added to the `TranslationFactory` class, because it will change how translation dates are created for the entire test suite but is only necessary for a few tests. Instead, you should use a method in your test class that creates the translation instances and passes in the dates you want to `TranslationFactory`.